build-support/bin/ci.sh

```diff
@@ -207,26 +199,13 @@ if [[ "${run_python:-false}" == "true" ]]; then
   fi
   start_travis_section "CoreTests" "Running core python tests${shard_desc}"
   (
-    ./pants.pex --tag='-integration' test.pytest --chroot \
-      --test-pytest-test-shard=${python_unit_shard} \
-      src/python:: tests/python:: -- ${PYTEST_PASSTHRU_ARGS}
+    ./pants.pex --tag='-integration' test.pytest \
+      --test-pytest-chroot --test-pytest-test-shard=${python_unit_shard} \
+      src/python:: tests/python:: contrib:: -- ${PYTEST_PASSTHRU_ARGS}
   ) || die "Core python test failure"
   end_travis_section
 fi
 
-if [[ "${run_contrib:-false}" == "true" ]]; then
-  if [[ "0/1" != "${python_contrib_shard}" ]]; then
-    shard_desc=" [shard ${python_contrib_shard}]"
-  fi
-  start_travis_section "ContribTests" "Running contrib python tests${shard_desc}"
-  (
-    ./pants.pex --exclude-target-regexp='.*/testprojects/.*' test.pytest \
```

Comment:
I think this `--exclude-target-regexp='.*/testprojects/.*'` is relevant and should be re-added to the two unit test calls above and below, since some contrib plugins define their own testprojects folder that may or may not be supposed to pass.